scripts/generate_jwt.py

```
@@ -0,0 +1,28 @@
+import argparse
+import time
+import jwt
+
+def generate_jwt(secret: str, aud: str = None, iss: str = None) -> str:
```

Ruff (0.13.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)